### DISCUSSION TITLE
Install the client

### DISCUSSION THREAD

**Discussion by Cleem05** [rustdesk/rustdesk#6153] [Q&A]
Hi, 

I'm trying to install the software (client) by GPO, everything is OK with a script but, when the software is launched, this message appear --> https://imgtr.ee/image/I7gTlv 

My question is, can i install the software without any manual action ( without click on " install "and " accept and install "  --> https://imgtr.ee/image/I7gTlv )

I already try with the option "--install " but the user have to accept manually and type an administrator password .. 

Thanks for any help,

**Comment by 21pages** [comment]
--silent-install, administrator permission still required.

**Reply by Cleem05** [reply]
--silent-install just launch the program and the user have to manually install it ( see here : https://imgtr.ee/image/I7gTlv )... 

The problem is, if the user close the RustDesk app, he must reboot the computer to get help, 

Thanks for your answer

**Comment by Cleem05** [comment]
-------------------------------------------------------------------
EDIT : 
I create a simple web server using apache2 and i push the rustdesk[...].exe inside, now i can use : 
```powershell
               Invoke-WebRequest "http://**myWebServer**/rustdesk-1.2.3-x86_64.exe" -Outfile "rustdesk.exe"
               Start-Process .\rustdesk.exe --silent-install
``` 

Now, how can i install the "rustdesk.exe" without typing the administrator password on the user computer ( gpo ? ) 

Thanks for any help,
